indecision-plaza/app/food-finder/food-finder.component.ts

```diff
@@ -11,18 +12,38 @@ export class FoodFinderComponent {
   public foodInfo: string = "";
   public chosenPlace: Business;
   public currentLocation: Location;
+
+  public categories: Array<string>;
+  public categoryMap: {[key: string]: string} = {};
+  public chosenCategory: string = "Restaurants";
   constructor(private foodSvc: FinderService) {
     enableLocationRequest();
+    this.chosenPlace = Business.makeEmpty();
+    this.categories = ["Bars","Coffee/Tea","Restaraunts","Breakfast","Fast Food"];
```

Comment:
Does Yelp's API offer a query to return categories?

Reply:
We looked at this, they have a JSON that we can use as a data depository, but I think this will not be entirely useful especially if we want to change the API soon. Thoughts @BassmanSolutions ?